Automattic/studio
# Studio: Disable buttons and tooltips when a site is syncing

## Related issues

Closes https://github.com/Automattic/dotcom-forge/issues/10023

## Proposed Changes

This PR adjusts the tooltips for the `Pull` and `Push` and `Disconnect` buttons as well as apply the styling for the buttons to be disabled when one of the sites is pulling or pushing:

<img width="706" alt="Screenshot 2024-12-04 at 4 51 17 PM" src="https://github.com/user-attachments/assets/10bbe424-1eb0-4439-9586-04bcb1329443">

<img width="458" alt="Screenshot 2024-12-04 at 4 29 17 PM" src="https://github.com/user-attachments/assets/a8fc6287-9b6d-4255-bcfa-e910dbc4960e">

## Testing Instructions



* Pull the changes from this branch
* Start the app with `npm start`
* Navigate to the `Sync` tab
* Ensure that you have at least 2 sites connected
* Start pull or push operation on one of the sites
* Observe that the tooltips over `Pull` and `Push` buttons indicate why they are disabled
* Confirm that the buttons are disabled and have styling of greyed out buttons
* Observe that `Disconnect` button is disabled 
* Confirm that you can see a tooltip with the information why it is disabled when you hover over

## Pre-merge Checklist
